packages/workbox-recipes/src/warmStrategyCache.ts

```diff
@@ -0,0 +1,30 @@
+import { Strategy } from 'workbox-strategies/src/Strategy';
+
+import './_version.js';
+
+export interface WarmStrategyCacheOptions {
+  paths: Array<string>;
+  strategy: Strategy;
+}
+
+// Give TypeScript the correct global.
+declare let self: ServiceWorkerGlobalScope;
+
+/**
+ * 
+ * @param {Object} options 
+ * @param {string[]} options.paths Paths to warm the strategy's cache with
+ * @param {Strategy} options.strategy Strategy to use
+ */
+function warmStrategyCache(options: WarmStrategyCacheOptions): void {
+  self.addEventListener('install', event => {
+    const warm = options.paths.map(path => options.strategy.handleAll({
```

Comment:
I think `done` (or `donePromises`) would be a more descriptive variable name than `warm`, especially since it's not entirely obvious from reading the code that the `[1]` index represents a `done` promise.